tests/fixtures/apps/demultiplexing/flowcell_runs/dragen/210428_A00689_0260_BHNC2FDSXY/SampleSheet.csv

```diff
@@ -0,0 +1,1194 @@
+[Data]
```

Comment:
if its for testing purposes, can it be truncated?

Reply:
I'm cleaning up the test data, it's going to be much much smaller